src/Compiler/ComponentTagCompiler.php

```diff
@@ -236,4 +241,27 @@ protected function parseAttributeBag(string $attributeString): string
 
         return (string) preg_replace($pattern, ' :attributes="$1"', $attributeString);
     }
+
+    /**
+     * Remove Twig comments from given attribute string
+     */
+    protected function pourOutCommentsFromAttributeBag(string $attributeString): string
+    {
+        $pattern = "/
+            (                                               # First capturing group
+                \{\#                                        # Match {# literally
+                    (                                       # Second capturing group
+                        [                                   # Match any character in the following list
```

Comment:
I do not know exactly but it should be somehow possible to use the positive/negative look ahead/behind this list. Because I want to say "there should not be another `{#` behind the list".

Reply:
Solved by https://github.com/lmc-eu/twigx-bundle/pull/19#discussion_r1081292817